Sources/UIAlertController.swift

```diff
@@ -10,7 +10,6 @@ import func Foundation.round
 
 public enum UIAlertControllerStyle {
     case actionSheet
-    case popover
```

Comment:
No idea why we had this. It actually does not exist on iOS.
https://developer.apple.com/documentation/uikit/uialertcontroller/style